app/controllers/RegularContributions.scala

```diff
@@ -78,7 +79,7 @@ class RegularContributions(
 
     result.fold(
       { error =>
-        logger.error(s"Failed to create new monthly contributor: $error")
```

Comment:
where does the error message go now, does it get logged at all?

Reply:
There's a bunch of more detailed logging here: https://github.com/guardian/support-frontend/pull/494/files/d7ea887a13fce661197bc37bef6d32173955f796#diff-ecaf18660141161c28efec09643d39a4

Reply:
... but it looks like I'd have dropped errors coming from [identityService](https://github.com/guardian/support-frontend/pull/494/files/d7ea887a13fce661197bc37bef6d32173955f796#diff-07e47d771fce3d75f900a978da262053L87) - nice catch!

Reply:
Fixed, thanks to your suggestion 👍